appium/webdriver/extensions/action_helpers.py

```diff
@@ -149,3 +149,87 @@ def flick(self, start_x, start_y, end_x, end_y):
             .release()
         action.perform()
         return self
+
+    def zoom_element(self, element):
+        """Zooms in on an element a certain amount
+
+        :Args:
+         - element - the element to zoom
+
+        :Usage:
+            driver.zoom(element)
+        """
+        multi_touch = MultiAction()
+        size=element.size
+        position = element.location
+        coordinate_x=position.get('x')
```

Comment:
We can expect they return x, y, with, height, so `position['x']` is preferred

Reply:
Please rebase to the latest on master branch and run below commands.

```
$python -m autopep8 -a -r --global-config .config-pep8 -i .
```